### DISCUSSION TITLE
Code changes aren't showing

### DISCUSSION THREAD

**Discussion by suss5** [vercel/community#2497] [Help] [answered]
### Summary

I used the Vercel KV demo code, and have made a number of changes (to the text showing for example)
The project was rebuilt (without errors) but it still shows the original text.
So nothing I edited, has actually gone through as a change

**Comment by amyegan** [comment, marked as answer]
Sorry you're having deployment problems. I think your repo is private, so I'm not able to see it. Can you walk me through the steps you took to deploy and redeploy the project?

**Reply by suss5** [reply]
Thanks, have made it Public :)

**Comment by suss5** [comment]
Update, its working now and showing the code changes.
Not sure what happened earlier.
Normally code changes are picked up within seconds, but now and again it doesn't happen and old app code seems to run. It may be an issue with the browser cache-ing previous code. I haven't fully got to the bottom of it, but its ok now!
